### ISSUE TITLE
publish --message removes package summary

### ISSUE THREAD

**Issue by tivac** [lerna/lerna#842]
## Expected Behavior
When using `lerna publish --message` I still want the commit body to contain all the tags that were created.

[This commit](https://github.com/tivac/modular-css/commit/4ac74b7e676219c9d8135d1db45ac8b74151162a) didn't use `--message` so it shows the created tags in the commit description.

## Current Behavior
The commit uses my custom message, but none of the updated packages are listed.

[This commit](https://github.com/tivac/modular-css/commit/1b811a7d5397026a2576904e57903c0de7869e84) shows the custom `--message` and how it prevents the tags from appearing in the commit message.

## Steps to Reproduce
1. `lerna publish --message="<message here>"`
2. Look at the most recent git commit, note that no tags are listed in the commit message

## Context
I use `validate-commit-msg` so in order for `lerna publish` to work I have to specify a custom `--message` param that will validate. I also want the details like what the default `publish` commit provides. When `--message` support was added in #460 it looks like this was the intent, but it's not working for me.

## Your Environment

| Executable | Version |
| ---: | :--- |
| `lerna --version` | 2.0.0-rc.5 |
| `npm --version`  | 4.6.1|
| `node --version` | 7.9.0 |

| OS | Version |
| --- | --- |
| Windows 10 | 14393 |

**Comment by evocateur** [comment]
@emmenko described what should happen when using `--independent` versioning mode [here](https://github.com/lerna/lerna/pull/460#discussion_r93754925).

For the default "fixed" versioning mode, should we preserve the original tag in the commit message somewhere? We might consider using [`npm version`](https://docs.npmjs.com/cli/version)'s `%s` placeholder, in that case.

```bash
# current default message
v1.0.0

# current --message "Release"
Release

# proposed --message "Release %s finally!"
Release v1.0.0 finally!
```

**Comment by tivac** [comment]
Yeah, @emmenko's comment is why I linked #460. His comment is exactly the behavior I was expecting and why I filed the issue.

For `fixed` versioning I think lerna should definitely support the `%s` placeholder like NPM. I even assumed it would [way back when](https://github.com/tivac/modular-css/blob/a5f4592157c5e5236c118c27538c428d5f0667d6/lerna.json) before I realized that lerna wouldn't use command args from `lerna.json`.
